src/run.py

```diff
@@ -0,0 +1,8 @@
+from app import app
+from os import environ
+
+if __name__ == "__main__":
+    HOST = "0.0.0.0"
```

Comment:
prefer lowercase here, uppercase is reserved for constants

Reply:
isn't host tech a constant

Reply:
global constants*

Reply:
i also don't super feel like this print message is necessary so if you remove it you can just inline these vars but either way is fine